EU4ToVic2Tests/HelpersTests/TechValuesTests.cpp

```diff
@@ -109,17 +105,16 @@ TEST(Helpers_TechValuesTests, getNormalizedArmyTechReturnsOneIfInitializedWithNo
 {
 	std::map<std::string, V2Country*> countries;
 	std::stringstream ideaEffectInput;
-	mappers::IdeaEffectMapper ideaEffectMapper(ideaEffectInput);
 
-	helpers::TechValues techValues(countries, ideaEffectMapper);
+	helpers::TechValues techValues(countries);
 
 	mockEU4Country country;
 	EXPECT_CALL(country, getAdmTech).WillOnce(testing::Return(32));
 	EXPECT_CALL(country, getMilTech).WillOnce(testing::Return(32));
 	std::map<std::string, int> nationalIdeas;
 	EXPECT_CALL(country, getNationalIdeas).WillOnce(testing::ReturnRef(nationalIdeas));
 
-	ASSERT_EQ(techValues.getNormalizedArmyTech(country, ideaEffectMapper), 1.0);
+	ASSERT_EQ(techValues.getNormalizedArmyTech(country), 0.0);
```

Comment:
I'm unclear how this could now be returning 0.0. Can you explain it to me?

Reply:
My mistake, function returns 1.0, and I've restored all tests to expect that. However, point is moot due to all tests in this file throwing SEH exceptions, cannot test if they actually work.